test/TestConfig/scripts/build_test.xml

```diff
@@ -124,6 +124,7 @@
 						<exclude name="TEST_*/build.xml" />
 						<exclude name="Panama/build.xml" />
 						<exclude name="OpenJ9_Jsr_292_API/build.xml" />
+						<exclude name="UnsafeTest/build.xml" />
```

Comment:
UnsafeTest is currently running on SE80 as sanity. This change remove this test from SE80 entirely. 

Also I saw the modification in test/TestConfig/scripts/testKitGen/makeGenTool/mkgen.pl. This whole test suite is excluded from current jcl version. 

I think we need to clarify two parts:
1. Changes in this PR, do they also run on Java8, or they only run on Java9?
2. Changes in this PR, do they only run on Java9 b181, but not b148?

Reply:
These tests are only intended to run on Java9 b181.This is also the only version the new library https://github.com/eclipse/openj9/pull/56 is set to run on.

In b174 all `compareAndSwap` methods in `jdk.internal.misc.Unsafe` were removed as reflected when the tests were re-enabled https://github.com/eclipse/openj9/pull/383. New methods `compareAndSet` were added which are not compatible with older jcls.

What configuration do you recommend?

Reply:
Thanks for the information. In this case, I would suggest a solution like this:

1. Add a JAVA_VERSION check to avoid excluding test from Java8. Change the structure in build_test.xml like this
```		<if>
			<equals arg1="${JAVA_VERSION}" arg2="SE80" />
			<then>
				<subant target="">
					<property name="build.dir" value="subant1.build" />
					<property name="not.overloaded" value="not.overloaded" />
					<property name="JAVA_VERSION" value="${JAVA_VERSION}" />
					<property name="compiler.javac" value="${JAVAC}" />
					<fileset dir="../../" includes="${build.list}">
						<exclude name="TEST_*/build.xml" />
						<exclude name="Panama/build.xml" />
						<include name="Valhalla/build.xml" />
						<exclude name="OpenJ9_Jsr_292_API/build.xml" />
					</fileset>
				</subant>
			</then>
			<elseif>
				<equals arg1="${JAVA_VERSION}" arg2="SE90" />
				<then>
					<if>
						<equals arg1="${JCL_VERSION}" arg2="latest" />
						<then>
							<subant target="">
								...
							</subant>
						</then>
						<elseif>
							<equals arg1="${JCL_VERSION}" arg2="Panama" />
							<then>
								<subant target="">
									...
								</subant>
							</then>
						</elseif>
						<elseif>
							<equals arg1="${JCL_VERSION}" arg2="Valhalla" />
							<then>
								<subant target="">
									...
								</subant>
							</then>
						</elseif>
						<else>
							<subant target="">
								...
							</subant>
						</else>
					</if>
				</then>
			</elseif>
		</if>```


2. We don't need to add this test project in mkgen.pl
As we do not care about the internal b148 Java9 anymore, any test that passes on OpenJ9 but failed on Java9 b148 should be excluded by using exclude file in the tooling repo.